app/views/course/question_assessments/_question_assessment.html.slim

```diff
@@ -13,27 +10,31 @@
             => fa_icon 'exclamation-triangle'.freeze,
                     title: t('.non_auto_gradable'),
                     class: 'text-warning'
-          .btn-group
+          - if can?(:manage, assessment)
             .btn-group
-              .dropdown
-                button.btn.btn-default.dropdown-toggle#duplicate-question-dropdown data-toggle='dropdown'
-                  i.fa.fa-copy
-                  '
-                  span.caret
-                ul.dropdown-menu.dropdown-menu-right
-                  - @question_duplication_dropdown_data.each_with_index do |tab_hash, index|
-                    - unless index == 0
-                      li.divider role='separator'
-                    li.dropdown-header = tab_hash[:title]
-                    - tab_hash[:assessments].each do |assessment_hash|
-                      li
-                        = link_to assessment_hash[:title],
-                                  duplicate_course_assessment_question_path( \
-                                    current_course, assessment, question, assessment_hash[:id]),
-                                  class: 'dropdown-item', method: :post, remote: true,
-                                  data: { confirm: t('.confirm_duplicate_question', destination: assessment_hash[:title]) }
-            = edit_button(edit_path)
-            = delete_button(delete_path)
+              .btn-group
+                .dropdown
+                  button.btn.btn-default.dropdown-toggle#duplicate-question-dropdown data-toggle='dropdown'
+                    i.fa.fa-copy
+                    '
+                    span.caret
+                  ul.dropdown-menu.dropdown-menu-right
+                    - @question_duplication_dropdown_data.each_with_index do |tab_hash, index|
+                      - unless index == 0
+                        li.divider role='separator'
+                      li.dropdown-header = tab_hash[:title]
+                      - tab_hash[:assessments].each do |assessment_hash|
+                        li
+                          = link_to assessment_hash[:title],
+                                    duplicate_course_assessment_question_path( \
+                                      current_course, assessment, question, assessment_hash[:id]),
+                                    class: 'dropdown-item', method: :post, remote: true,
+                                    data: { confirm: t('.confirm_duplicate_question', destination: assessment_hash[:title]) }
+
+              - edit_path = url_for([:edit, current_course, assessment, question.specific])
```

Comment:
Unlike surveys, not all the question info is available on the attempt page. I think there is a need to allow observers to (per)view the private and evaluation test cases, staff-only notes etc.

For observers, what do you think if we have a dropdown with 'Show Question' that links to the edit page? The page can show the disabled edit form.

Reply:
I was hoping to delay this to a separate PR (if necessary) because the observers are able to view the information when attempting the assessment 😝(except staff-only notes). 

Not sure if a disabled edit form is the way to go though, from what I understand the disabled forms only displays the text but does not allow selection - will observers need to somehow copy paste some of the text?

Reply:
Will also delay the showing of private and evaluation test cases to the next PR.